libraries/AC_PrecLand/AC_PrecLand_Companion.h

```diff
@@ -37,7 +39,8 @@ class AC_PrecLand_Companion : public AC_PrecLand_Backend
 private:
 
     // output from camera
-    Vector2f            _bf_angle_to_target;    // last body-frame angle to target
+    MAV_FRAME           _frame;                 // what frame of reference is our sensor reporting in?
+    Vector2f            _angle_to_target;       // last body-frame angle to target
```

Comment:
Looks good. You may want to change the comments to reflect the change in variable names